produce.go

```diff
@@ -25,7 +46,7 @@ type ProduceRequest struct {
 	Partition int
 
 	// The level of required acknowledgements to ask the kafka broker for.
-	RequiredAcks int
+	RequiredAcks RequiredAcks
```

Comment:
Interesting. I didn't even know these were code for none/one/all, I thought I was specifying exactly how many acks I wanted.